gen-dirs.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+set -e
+cd templates
+FILES=$(find -follow)
+VERSIONS="9.4 9.5 9.6"
```

Comment:
VERSIONS should get passed to the script from the Makefile, so that the list can be maintained in one place.